code/datums/ai/ai_holo.dm

```diff
@@ -11,7 +11,12 @@
 
 
 /decl/ai_holo/proc/may_be_used_by_ai(var/mob/living/silicon/ai/AI)
-	return AI.is_traitor()
+	if(requires_malf)
```

Comment:
`return !requires_malf || AI.malfunctioning`? Current code will lock malf AIs to the malf icons only.